source/Camera/Basic/CaptureTutorial.md

```diff
@@ -110,7 +110,7 @@ We may choose to configure settings manually. For more information about what ea
 We can create settings for a single capture ([go to source][settings-url]).
 ```cpp
 const auto settings = Zivid::Settings{
-    Zivid::Settings::Frames{
+    Zivid::Settings::Acquisitions{
         Zivid::Settings::Acquisition{ Zivid::Settings::Acquisition::Aperture{ 5.66 },
                                 Zivid::Settings::Acquisition::ExposureTime{ std::chrono::microseconds{ 8333 } },
     Zivid::Settings::Processing::Filters::Outlier::Removal::Enabled::yes,
```

Comment:
Under "Multi Acquisition HDR" below, since settings is `const`, doing `settings.acquisitions().emplaceBack(acquisitionSettings);` won't work. So you need to remove `const` from `const auto settings`.

Reply:
Done